android/.idea/.name

```diff
@@ -0,0 +1 @@
+TF Segement + Stylize
```

Comment:
Typo - "Segement" should be "Segment".  Later we can perhaps come up with a better app name.

Reply:
🙈 Missed this, thanks - have fixed it and pushed.